index.html

```diff
@@ -52,39 +258,40 @@ <h2>О нас</h2>
 
     <section>
       <h2>Контакты</h2>
-        <p>Вы можете забрать товар сами, заехав в офис. Заодно, вы сможете <br>
-          проверить работоспособность покупки. Всякое бывает.</p><br>
-          <img src="#" alt="Карта проезда">
-          <a href="">Напишите нам</a>
+      <p>Вы можете забрать товар сами, заехав в офис. Заодно, вы сможете <br>
+        проверить работоспособность покупки. Всякое бывает.</p><br>
+      <img src="#" width="1" height="1" alt="Карта проезда">
```

Comment:
Ссылка с изображением:
```suggestion
      <a href=""><img src="#" width="1" height="1" alt="Карта проезда"></a>
```

Об этом сказано в [ТЗ](https://up.htmlacademy.ru/htmlcss/28/project/device#device-3-8).